docs/LayoutSpecs.json

```diff
@@ -49,10 +49,10 @@
                   "path": "DSCore.List.IsRectangular"
                 },
                 {
-                  "path": "TrueForAll"
+                  "path": "List.TrueForAll"
```

Comment:
Does this reflect the latest `LayoutSpecs.json` in Dynamo repo? What about `RawTypeData.json`? We need to make sure `librarie.js` has the latest from Dynamo repo for this period (although it doesn't quite matter).